src/ZendCodingStandard/Sniffs/Commenting/VariableCommentSniff.php

```diff
@@ -0,0 +1,126 @@
+
+        $foundVar = null;
+        foreach ($tokens[$commentStart]['comment_tags'] as $tag) {
+            if ($tokens[$tag]['content'] === '@var') {
+                if ($foundVar !== null) {
+                    $error = 'Only one @var tag is allowed in a member variable comment';
+                    $phpcsFile->addError($error, $tag, 'DuplicateVar');
+                } else {
+                    $foundVar = $tag;
+                }
+            } elseif ($tokens[$tag]['content'] === '@see') {
+                // Make sure the tag isn't empty.
+                $string = $phpcsFile->findNext(T_DOC_COMMENT_STRING, $tag, $commentEnd);
+                if ($string === false || $tokens[$string]['line'] !== $tokens[$tag]['line']) {
+                    $error = 'Content missing for @see tag in member variable comment';
+                    $phpcsFile->addError($error, $tag, 'EmptySees');
+                }
+            } else {
+                $error = '%s tag is not allowed in member variable comment';
+                $data = [$tokens[$tag]['content']];
+                $phpcsFile->addWarning($error, $tag, 'TagNotAllowed', $data);
+            }
+        }
+
+        // The @var tag is the only one we require.
+        if ($foundVar === null) {
+            $error = 'Missing @var tag in member variable comment';
+            $phpcsFile->addError($error, $commentEnd, 'MissingVar');
+            return;
+        }
+
+        $firstTag = $tokens[$commentStart]['comment_tags'][0];
+        if ($foundVar !== null && $tokens[$firstTag]['content'] !== '@var') {
+            $error = 'The @var tag must be the first tag in a member variable comment';
+            $phpcsFile->addError($error, $foundVar, 'VarOrder');
+        }
+
+        // Make sure the tag isn't empty and has the correct padding.
+        $string = $phpcsFile->findNext(T_DOC_COMMENT_STRING, $foundVar, $commentEnd);
+        if ($string === false || $tokens[$string]['line'] !== $tokens[$foundVar]['line']) {
+            $error = 'Content missing for @var tag in member variable comment';
+            $phpcsFile->addError($error, $foundVar, 'EmptyVar');
+            return;
+        }
+
+        $varType = $tokens[$foundVar + 2]['content'];
+        // changed to use bool and int instead of boolean and integer accordingly
+        $suggestedType = CodingStandard::suggestType($varType);
+        if ($varType !== $suggestedType) {
+            $error = 'Expected "%s" but found "%s" for @var tag in member variable comment';
+            $data = [
+                $suggestedType,
+                $varType,
+            ];
+            $fix = $phpcsFile->addFixableError($error, $foundVar + 2, 'IncorrectVarType', $data);
+
+            if ($fix) {
+                $phpcsFile->fixer->replaceToken($foundVar + 2, $suggestedType);
+            }
+        }
+    }
+
+    /**
+     * @inheritDoc
+     */
+    protected function processVariable(File $phpcsFile, $stackPtr)
+    {
+    }
```

Comment:
Comment added. It was not here, because this sniff is copied form PHP_CodeSniffer with changes described on top of the file. I think this sniff should be rewritten at all (and we need make sure that is working correctly with our new standards for PHP 7.1+).